catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -213,12 +211,8 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
 
 		@Override
 		public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
-			ConnectivityManager cm = (ConnectivityManager) getBaseContext().getSystemService(Context.CONNECTIVITY_SERVICE);
-			NetworkInfo activeNetwork = cm.getActiveNetworkInfo();
-			boolean isConnected = activeNetwork != null && activeNetwork.isConnectedOrConnecting();
```

Comment:
This is a small behavior change. Now it only depends on `isConnected` (not `isConnectedOrConnecting`), if the additional info `Check your Internet connection` is shown. IMO in both cases (not connected or connecting), it's fine to show this additional info, because in both cases you don't have a connection (to the internet).

Also [`isConnectedOrConnecting`](https://developer.android.com/reference/android/net/NetworkInfo.html#isConnectedOrConnecting())
> This method was deprecated in API level 28.

(but for other reasons)